cmd/kubermatic-api/options.go

```diff
@@ -82,7 +81,6 @@ func newServerRunOptions() (serverRunOptions, error) {
 	s.log.AddFlags(flag.CommandLine)
 
 	flag.StringVar(&s.listenAddress, "address", ":8080", "The address to listen on")
-	flag.StringVar(&s.kubeconfig, "kubeconfig", "", "Path to the kubeconfig.")
```

Comment:
How would I now run the API locally? Is it all based on the `KUBECONFIG` env variable now?

Reply:
the `kubeconfig` flag is still there, I had to remove it because it is defined by controller-runtime already.